backend/db/migrate/20230608080932_rename_api_entreprise_scopes_in_enrollments.rb

```diff
@@ -0,0 +1,103 @@
+class RenameApiEntrepriseScopesInEnrollments < ActiveRecord::Migration[7.0]
+  def up
+    Enrollment.find_each do |enrollment|
```

Comment:
On ne veut migrer que les demandes DataPass d'API Entreprise

```suggestion
    Enrollment.where(target_api: 'api_entreprise').find_each do |enrollment|
```